docs/guides/headless-gatsby-tutorial.md

```diff
@@ -4,99 +4,127 @@ title: Creating a blog with Headless CMS + Gatsby
 sidebar_label: Creating a blog with Headless CMS + Gatsby
 ---
 
-In this tutorial you will learn how to use Gatsby with our Headless CMS.
+In this tutorial we will learn how to use [Gatsby](https://www.gatsbyjs.org/) with the [Webiny Headless CMS](/docs/webiny-apps/headless-cms/features/content-modeling).
 
-[Here is a sample gatsby blog using Webiny's Headless CMS](https://github.com/webiny/webiny-examples/blob/master/headlesscms-gatsby)   
+> All of the code shown in this tutorial is also hosted in our [GitHub](https://github.com/webiny/webiny-examples/blob/master/headlesscms-nextjs) repository.) repository.
```

Comment:
repository.) repository.
 just one repository is fine

Reply:
fixed